examples/tm1638/tm1638.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+/*
+	Example works with TM1638 with eight 7-segment indicators at odd addresses
+	and eight LEDs at even addresses. Also eight buttons connected to first scan line.
+	Configuration implemented by board MDU1093.
+*/
+
+import (
+	"machine"
+	"time"
+
+	"tinygo.org/x/drivers/tm1638"
+)
+
+func main() {
+	// codes of nombers from 0 to 7 at odd indexes
+	toShow := []uint8{0x3f, 1, 0x06, 0, 0x5b, 1, 0x4f, 0, 0x66, 1, 0x6d, 0, 0x7d, 1, 0x07}
```

Comment:
That's an odd sequence of bytes, it doesn't look at all like ASCII text. I think you should explain what this means, and/or add documentation to `WriteArray` to explain what this array really means (if it isn't ASCII text).

Reply:
I will add explanation.